source/locale/en/symbols.dic

```diff
@@ -61,7 +61,7 @@ $	dollar	all	norep
 ,	comma	all	always
 、	ideographic comma	all	always
 ،	arabic comma	all	always
--	dash	most
+-	dash	most	norep
```

Comment:
Some synthesizers make a short pause when used as a punctuation such as in the following sentence:
He says - and I fully agree with him - that no discrimination should be tolerated here.

If we want to keep this pause at high symbol level, I recommend to set the `preserve` to always, as it is done for other punctuation that change the prosody such as comma.

```suggestion
-	dash	most	always
```

Regarding in-word dash, I have made tests and have not found any case for which it causes an issue with OneCore and eSpeak. In case there is an issue with in-word dashes, we may define a complex symbol for it to handle it differently as it is done with in-word tick.